lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -72,6 +72,12 @@
       - Dictionary of IPAM options.
     default: null
 
+  enable_ipv6:
+    description:
+      - Enable IPv6 on the network.
+    default: false
```

Comment:
Instead of `default: false` this should have no default, and only forward the `enable_ipv6` value to the docker API if the user manually sets this, otherwise whatever the docker daemon is set to default to (currently False, I think) will be used.

Reply:
I've removed the default option, however the [underlying API](https://docker-py.readthedocs.io/en/stable/api.html#docker.api.network.NetworkApiMixin.create_network) still give a default value of `False`.